src/app/pages/ArticlePage/Byline/index.tsx

```diff
@@ -104,6 +109,17 @@ const Byline = ({ blocks, children }: PropsWithChildren<Props>) => {
         {articleInformationTranslated}
       </VisuallyHiddenText>
       <ul css={BylineCss.bylineList} role="list">
+        {image && (
+          <li
+            css={
+              isRtl
+                ? [BylineCss.imageRtl, BylineCss.Image]
+                : [BylineCss.imageLtr, BylineCss.Image]
+            }
+          >
+            <img css={BylineCss.imageSource} src={image} alt="" />
```

Comment:
Does this need to work in AMP? If so, should we use the new `Image` component?

Reply:
Yeah it will do. I'll implement the new `Image` component on the main Bylines branch